### PR TITLE
[ALLUXIO-1764] Remove unused constant alluxio.AlluxioURI#WINDOWS

### PR DESCRIPTION
**Pull request by yuanqingyecmu**

### CODE REVIEW

**Comment by yupeng9** [comment]
okay to test

File: core/common/src/main/java/alluxio/AlluxioURI.java [hunk 1]
```diff
@@ -37,7 +37,12 @@
 
   public static final AlluxioURI EMPTY_URI = new AlluxioURI("");
 
-  private static final boolean WINDOWS = OSUtils.isWindows();
+  /**
+   * Commented by Yuanqing Ye (yuanqiny@andrew.cmu.edu)
+   * For the task.
+   * The WINDOWS boolean variable is unused, so I comment this line of code.
+   */
+//  private static final boolean WINDOWS = OSUtils.isWindows();
```
Comment (yupeng9) [inline core/common/src/main/java/alluxio/AlluxioURI.java]: This variable can be removed, and therefore not necessary to be commented out.

**Comment by apc999** [comment]
Jenkins, test this please

**Comment by yupeng9** [comment]
LGTM. Thanks for the improvement.

**Comment by yuanqingyecmu** [comment]
Thank you all!
Note that in the later commit, I removed the unused import
alluxio.util.OSUtils  to pass the style check, since it shows error after I
removed the unused WINDOW variable. Hope that I didn't mess things up.

Best,
Yuanqing

**Comment by yupeng9** [comment]
No worries. Everything built.

**Comment by yuanqingyecmu** [comment]
OK. Thanks!

**Pull request merged**